bin/hack-sound-server.in

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+import logging
+import gi
+import os
+import signal
+import sys
+gi.require_version('Gst', '1.0')
+gi.require_version('GLib', '2.0')
+from gi.repository import Gst
+from gi.repository import GLib
+
+
+LIBDIR = "@LIBDIR@"
+_logger = logging.getLogger(__name__)
+
+
+def _add_hack_sound_server_path():
+    path = os.path.join(LIBDIR, "hack-sound-server", "python")
+    if path not in sys.path:
+        sys.path.append(path)
+
+
+def _run_server():
+    from hack_sound_server.server import HackSoundPlayer
+    from hack_sound_server.server import HackSoundServer
+    from hack_sound_server.utils.metadata import read_and_parse_metadata
```

Comment:
Why do you import these here?

Reply:
Because `hack_sound_server` isn't available until its path is added to `sys.path`. Putting this at the beginning of the file wouldn't work.